pkg/object/apirule.go

```diff
@@ -166,3 +166,12 @@ func WithRules(certsURL string, subs []eventingv1alpha2.Subscription, svc apigat
 		r.Spec.Rules = rules
 	}
 }
+
+// StringsToMethods converts a slice of strings into a slice of HttpMethod as defined by api-gateway.
+func StringsToMethods(methods []string) []apigatewayv1beta1.HttpMethod {
```

Comment:
Can you add a tiny unit test for this method?

Reply:
sure